client/src/main/java/org/apache/rocketmq/client/impl/MQClientAPIImpl.java

```diff
@@ -381,6 +381,11 @@ public void createTopic(final String addr, final String defaultTopic, final Topi
         throws RemotingException, MQBrokerException, InterruptedException, MQClientException {
         Validators.checkTopicConfig(topicConfig);
 
+        if (getTopicListFromNameServer(timeoutMillis).getTopicList().contains(topicConfig.getTopicName())) {
+
```

Comment:
redundant line

Reply:
Thank you @joeCarf I will change.  Actually, i want to fetch all the topics present in the cluster but unable to find out. I checked this approach but this is not working, and causing a test case to fail. When i debug the test case. I found the getTopicList() method returning null response.